packages/@tinacms/toolkit/src/packages/fields/plugins/MdxFieldPlugin/updated/plate/plugins/core/autoformat/autoformatBlocks.ts

```diff
@@ -109,4 +110,17 @@ export const autoformatBlocks: AutoformatRule[] = [
       })
     },
   },
+  // that probably should not be here...
+  {
+    mode: 'inline',
```

Comment:
I was also getting a type error here. What promoted you to use `mode: 'inline'`

Reply:
just poking around TBH... semantically speaking `inline` here seems more adequate since it is not like a paragraph (and 
contains something / has children).  
But then... It's probably not the place it should be.